packages/cbl_flutter/lib/cbl_flutter.dart

```diff
@@ -29,7 +29,10 @@ class CouchbaseLiteFlutter {
 Future<InitContext> _context() async {
   final directories = await Future.wait([
     getApplicationSupportDirectory(),
-    getExternalStorageDirectory().onError<UnsupportedError>((_, __) => null),
+    if (!Platform.isIOS)
```

Comment:
Let's check for `Platform.isAndroid`. That's the only platform where this function is supported.

Reply:
@blaugold done